asa-manager/scripts/docker/run.cmd

```diff
@@ -21,15 +21,21 @@ IF %ERRORLEVEL% NEQ 0 GOTO FAIL
 :: Start the application
 echo Starting ASA manager ...
 docker run -it -p 9024:9024 ^
-    -e PCS_TELEMETRY_WEBSERVICE_URL ^
-    -e PCS_CONFIG_WEBSERVICE_URL ^
-    -e PCS_IOTHUBMANAGER_WEBSERVICE_URL ^
-    -e PCS_TELEMETRY_DOCUMENTDB_CONNSTRING ^
-    -e PCS_EVENTHUB_CONNSTRING ^
-    -e PCS_EVENTHUB_NAME ^
+    -e PCS_TELEMETRY_WEBSERVICE_URL=http://host.docker.internal:9004/v1 ^
```

Comment:
Why are these environment variable changes needed?

Reply:
The list of environment variables was different in asa-manager dotnet versus java. So I made it consistent.

Reply:
Got it.